music/src/piano_genie/model.ts

```diff
@@ -2,7 +2,7 @@
  * Core implementation for Piano Genie model.
  *
  * @license
- * Copyright 2018 Google Inc. All Rights Reserved.
+ * Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
this doesn't actually need to change btw :)

Reply:
Thanks! That's good to know :)